public/javascripts/SVLabel/src/SVLabel/Main.js

```diff
@@ -446,14 +444,6 @@ function Main (params) {
         svl.ui.statusMessage.title = $("#current-status-title");
         svl.ui.statusMessage.description = $("#current-status-description");
 
-        // OverlayMessage.
-        svl.ui.overlayMessage = {};
-        svl.ui.overlayMessage.holder = $("#overlay-message-holder");
```

Comment:
Looks like the HTML element associated with this is still in `explore.scala.html`